src/mlpack/methods/ann/rnn_impl.hpp

```diff
@@ -341,7 +346,12 @@ EvaluateWithGradient(const arma::mat& /* parameters */,
   for (size_t seqNum = 0; seqNum < rho; ++seqNum)
   {
     currentGradient.zeros();
-
+    size_t x=0;
+    x = (rho>r_size)?r_size:rho;
```

Comment:
Again , wrap operator with spaces

Also we could avoid creating the extra variable

Reply:
I tried it but we cannot use ternary operator in fucntion call